tooling/format-file-header.js

```diff
@@ -0,0 +1,184 @@
+const path = require("path");
+const fs = require("fs");
+
+// When --write is set, files will be written in place
+// Elsewise it only prints outdated files
+const doWrite = process.argv.includes("--write");
+
+const allFiles = new Set();
+const findFiles = p => {
+	const s = fs.statSync(p);
+	if (s.isDirectory()) {
+		for (const name of fs.readdirSync(p)) {
+			if (name.startsWith(".")) continue;
+			findFiles(path.join(p, name));
+		}
+	} else if (s.isFile()) {
+		allFiles.add(p);
+	}
+};
+findFiles(path.resolve(__dirname, "../lib"));
+
+let canUpdateWithWrite = false;
+
+const sortImport = (a, b) => {
+	if (!a.key.startsWith(".") && b.key.startsWith(".")) return -1;
+	if (a.key.startsWith(".") && !b.key.startsWith(".")) return 1;
+	if (a.key < b.key) return -1;
+	if (a.key > b.key) return 1;
+	return 0;
+};
+
+const execToArray = (content, regexp) => {
+	const items = [];
+	let match = regexp.exec(content);
+	while (match) {
+		items.push({
+			content: match[0],
+			key: match[1] + match[2]
+		});
+		match = regexp.exec(content);
+	}
+	return items;
+};
+
+const schema = [
+	{
+		title: "license comment",
+		regexp: /\/\*\n\s*MIT License http:\/\/www\.opensource\.org\/licenses\/mit-license\.php\n\s*(?:(Authors? .+)\n)?\s*\*\/\n/g,
+		updateMessage: "update the license comment",
+		update(content, author) {
+			return (
+				[
+					"/*",
+					"\tMIT License http://www.opensource.org/licenses/mit-license.php",
```

Comment:
I think the following banner would be better:

```js
/**
 * @license MIT
 * @author Tobias Koppers @sokra
 */
```

Or the inlined version:

```js
/**
 * @license
 * This source code is licensed under the MIT license.
 * See https://opensource.org/licenses/mit-license
 * 
 * @author Tobias Koppers @sokra
 */
```

Reply:
yep this can be considered. Although we need to check if this affects bundle size, because `@license` comments are kept while minimizing. We shouldn't have them in the buildins.

I leave this for a separate PR. I only want to make it consistent in this PR and this is the existing comment.